packages/kotti-ui/source/kotti-field-number/constants.ts

```diff
@@ -23,22 +25,28 @@ export const KOTTI_FIELD_NUMBER_PROPS = {
 	value: { default: null, type: Number },
 }
 
-export const DECIMAL_PLACES = 3
-// FIXME: should we consider manually setting the decimal separator using the translation context
-export const DECIMAL_SEPARATOR = (1.1).toLocaleString().replace(/\d/g, '')
+/**
+ * RegExp character set for use within other regular expressions
+ */
+export const DECIMAL_SEPARATORS_CHARACTER_SET = [
+	'[',
+	...Object.values(DecimalSeparator).map((x) => `\\${x}`),
```

Comment:
I'm not sure if escaping is necessary in character sets

Reply:
changed in latest